packages/dso-toolkit/src/components/tabs/tabs.models.ts

```diff
@@ -1,10 +1,17 @@
 export interface Tabs<TemplateFnReturnType> {
-  items: TabsItem<TemplateFnReturnType>[];
+  items: TabsItem[];
+  content: TemplateFnReturnType | string;
 }
 
-export interface TabsItem<TemplateFnReturnType> {
-  label: string;
-  id: string;
+export interface TabsItem {
+  label?: string;
+  identifier: string;
+  href?: string;
   modifiers?: "active" | "disabled";
```

Comment:
Ik ben ff kwijt of we hier over hebben gesproken: Gingen we dit niet ontmantelen?

Reply:
Ik weet het niet meer.